lib/commands/file-movement.js

```diff
@@ -210,9 +229,13 @@ async function pushFileToRealDevice (device, remotePath, base64Data) {
     await mkdirpDevice(service, path.dirname(relativePath));
     const stream = await service.createWriteStream(relativePath, { autoClose: true });
     stream.write(Buffer.from(base64Data, 'base64'));
-    const closeEvent = new B((resolve) => stream.on('close', resolve));
+    const closeEvent = new B((resolve) => stream.on('close', resolve)).timeout(IO_TIMEOUT);
     stream.destroy();
-    await closeEvent;
+    try {
+      await closeEvent;
+    } catch (e) {
+      throw new Error(`Couldnt push the file within the given timeout ${IO_TIMEOUT}ms`);
```

Comment:
s/Couldnt/Couldn't/ ?

Reply:
When I tried to upload a big file, 50mb, then could observe below messages on appium server.
Should we handle them? (just a confirmation/)

`Failed to receive any data within the timeout: 10000` happened 

```
JQCK4WucYrQMzUMwCSeNQwyGw...
[debug] [XCUITest] Executing command 'pushFile'
[debug] [XCUITest] Parsed container type: documents
(node:76465) UnhandledPromiseRejectionWarning: Error: Failed to receive any data within the timeout: 10000
    at Timeout.setTimeout (/Users/kazu/GitHub/appium-xcuitest-driver/node_modules/appium-ios-device/lib/afc/index.js:255:31)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
(node:76465) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:76465) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
(node:76465) UnhandledPromiseRejectionWarning: Error: Failed to receive any data within the timeout: 10000
    at Timeout.setTimeout (/Users/kazu/GitHub/appium-xcuitest-driver/node_modules/appium-ios-device/lib/afc/index.js:255:31)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
(node:76465) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
[debug] [W3C (42d14ed6)] Encountered internal error running command: Error: Couldnt push the file within the given timeout 30000ms
[debug] [W3C (42d14ed6)]     at pushFileToRealDevice (/Users/kazu/GitHub/appium-xcuitest-driver/lib/commands/file-movement.js:237:13)
[HTTP] <-- POST /wd/hub/session/42d14ed6-3f16-4281-acce-554ad11aad3a/appium/device/push_file 500 32452 ms - 587
```